odl/contrib/fom/supervised.py

```diff
@@ -473,20 +494,41 @@ def ssim(data, ground_truth,
         Input data to compare to the ground truth.
     ground_truth : `FnBaseVector`
         Reference to compare ``data`` to.
-    size : odd int
+    size : odd int, optional
         Size in elements per axis of the Gaussian window that is used
         for all smoothing operations.
-    sigma : positive float
+    sigma : positive float, optional
         Width of the Gaussian function used for smoothing.
-    K1, K2 : positive float
-        TODO
-    dynamic_range : nonnegative float
-        TODO
+    K1, K2 : positive float, optional
+        Small constants to stabilize the result. See [Wan+2004] for details.
+    dynamic_range : nonnegative float, optional
+        Difference between the maximum and minimum value that the pixels
+        can attain. Use 255 if pixel range is :math:`[0, 255]` and 1 if
+        it is :math:`[0, 1]`. Default: `None`, obtain maximum and minimum
+        from the ground truth.
+    normalized  : bool, optional
+        Boolean flag to switch between unormalized and normalized FOM.
+        See `Notes` for details.
+    force_lower_is_better : bool, optional
```

Comment:
I'm not sure I like this flag. What's the reasoning behind it? I assume the goal of this and `normalized` is to have a set of FOMs that are immediately comparable.

I'm a bit worried about adding a parameter that does nothing in a lot of FOMs (all that end in `error` basically). For those we should state the purpose of the flag, namely that it does nothing in this case and exists to make all FOMs directly comparable, in particular the "error" type with the "goodness-of-fit" type.